### ISSUE TITLE
Cannot `stringify` without `Compiler`

### ISSUE THREAD

**Issue by AlexAegis** [remcohaszing/unified-consistency#1]
I got this error since I updated unified-prettier to 2.0.0 :(

```sh
docs/01-blog-draft.md
  1:1  error  TypeError: Cannot `stringify` without `Compiler`
    at assertCompiler (file:///home/alex/git/@aa/js-tooling/node_modules/.pnpm/unified@10.1.2/node_modules/unified/lib/index.js:520:11)
    at Function.stringify (file:///home/alex/git/@aa/js-tooling/node_modules/.pnpm/unified@10.1.2/node_modules/unified/lib/index.js:281:5)
    at file:///home/alex/git/@aa/js-tooling/node_modules/.pnpm/unified-consistency@1.0.1/node_modules/unified-consistency/index.js:15:39
    at wrapped (file:///home/alex/git/@aa/js-tooling/node_modules/.pnpm/trough@2.1.0/node_modules/trough/index.js:115:27)
    at next (file:///home/alex/git/@aa/js-tooling/node_modules/.pnpm/trough@2.1.0/node_modules/trough/index.js:65:23)
    at Object.run (file:///home/alex/git/@aa/js-tooling/node_modules/.pnpm/trough@2.1.0/node_modules/trough/index.js:36:5)
    at executor (file:///home/alex/git/@aa/js-tooling/node_modules/.pnpm/unified@10.1.2/node_modules/unified/lib/index.js:321:20)
    at Function.run (file:///home/alex/git/@aa/js-tooling/node_modules/.pnpm/unified@10.1.2/node_modules/unified/lib/index.js:312:5)
    at transform (file:///home/alex/git/@aa/js-tooling/node_modules/.pnpm/unified-engine@10.1.0/node_modules/unified-engine/lib/file-pipeline/transform.js:25:23)
    at wrapped (file:///home/alex/git/@aa/js-tooling/node_modules/.pnpm/trough@2.1.0/node_modules/trough/index.js:115:27)
```

**Comment by remcohaszing** [comment]
Recently unified 11 was released, and I immediately upgraded `unified-prettier`. Most of the ecosystem isn’t ready yet, including remark and `unified-consistency`. You ran into this issue with `unified-consistency` first by coincidence.

I will update this package once remark is updated (tests depend on remark). Once that’s fixed, there’s still risk of other plugins being out-of-date. It will all be updated soon, but this takes time.

**Comment by remcohaszing** [comment]
This was fixed in [v2.0.0](https://github.com/remcohaszing/unified-consistency/releases/tag/v2.0.0).
